[["C:/Users/chudds/AppData/Local/Packages/PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0/LocalCache/local-packages/Python311/site-packages/imdb/__init__.py",[false,[["IMDbError",true,2,["IMDbError","C:/Users/chudds/AppData/Local/Packages/PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0/LocalCache/local-packages/Python311/site-packages/imdb/_exceptions.py",5]],["IMDb",true,12,null],["Cinemagoer",true,13,null],["available_access_systems",true,12,null]]]]]